### ISSUE TITLE
How to obtain vectors all of the nodes?

### ISSUE THREAD

**Issue by myrainbowandsky** [louisabraham/fastnode2vec#20]
The code is 
`
graph = Graph(_lst, directed=True, weighted=True)`

`model = Node2Vec(_graph, dim=300, walk_length=100, context=10, p=2.0, q=0.5, workers=-1)`
    
`model.train(epochs=epochs)`

I have 10,000 nodes. When I check:
 
`model.index_to_key`, 
there are only 502 nodes. 
Why is that? 
How to set parameters so I can get the vectors of all nodes?

**Comment by louisabraham** [comment]
What makes you think you have 10k nodes?

**Comment by louisabraham** [comment]
Closing this now.
